model/DataStore/PersistDataService.php

```diff
@@ -164,4 +145,33 @@ private function getFileSystemManager(): FileSystemService
     {
         return $this->getServiceLocator()->get(FileSystemService::SERVICE_ID);
     }
+
+    private function addMetaDataFile(ZipArchive $zipFile, string $fileNameToAdd, string $content): bool
```

Comment:
You never use a return value so may be make it void?

Reply:
done